open-weather-connector-test/src_test/com/axonivy/connector/openweather/test/it/BaseProcessTest.java

```diff
@@ -0,0 +1,74 @@
+package com.axonivy.connector.openweather.test.it;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.BeforeEach;
+
+import ch.ivyteam.ivy.application.IApplication;
+import ch.ivyteam.ivy.bpm.engine.client.BpmClient;
+import ch.ivyteam.ivy.bpm.engine.client.element.BpmProcess;
+import ch.ivyteam.ivy.bpm.engine.client.sub.SubRequestBuilder;
+import ch.ivyteam.ivy.bpm.exec.client.IvyProcessTest;
+import ch.ivyteam.ivy.environment.AppFixture;
+import ch.ivyteam.ivy.rest.client.RestClient;
+import ch.ivyteam.ivy.rest.client.RestClients;
+
+@IvyProcessTest(enableWebServer = true)
+public class BaseProcessTest {
```

Comment:
junit5 should no longer be used with base-classes: normally you'd rather pack such code as in this class in a junit5-extension.
And then add it to each test via @RegisterExtension
![junit5-ext](https://github.com/user-attachments/assets/eec0f9c4-ad25-4f18-9e61-d0e7b524049b)